specs/core/0_fork-choice.md

```diff
@@ -130,13 +144,43 @@ def get_head(store: Store) -> Hash:
         head = max(children, key=lambda root: (get_latest_attesting_balance(store, root), root))
 ```
 
+#### `should_update_justified_checkpoint`
+
+```python
+def should_update_justified_checkpoint(store: Store, justified_checkpoint: Checkpoint) -> bool:
+    if get_current_slot(store) % SLOTS_PER_EPOCH < SAFE_SLOTS_TO_UPDATE_JUSTIFIED:
```

Comment:
We don't make the assumption that slots are epoch aligned anywhere else in the spec (afaik). Maybe we add a helper func for slots since epoch start? There exists a chance that epoch length changes in a fork, like with v0.9

Reply:
But we do make the assumption that epochs are `SLOTS_PER_EPOCH` within helpers, but I suppose the internals there can be easily swapped out.

Will add a helper